bindings/cpu_profiler.cc

```diff
@@ -275,7 +397,7 @@ static void GetFrameModule(const std::string& abs_path, std::string& module) {
   if (module[0] == '.') {
     module = module.substr(1, std::string::npos);
   }
-}
```

Comment:
nit: there are a few odd formatting issues like this one, maybe worth running an auto-formatter on this code